common/containers/Tabs/Swap/components/ReceivingAddress.tsx

```diff
@@ -28,33 +28,29 @@ export interface ActionProps {
   bityOrderCreateRequestedSwap: TBityOrderCreateRequestedSwap;
 }
 
-export default class ReceivingAddress extends Component<
-  StateProps & ActionProps,
-  {}
-> {
-  public onChangeDestinationAddress = (
-    event: React.SyntheticEvent<HTMLInputElement>
-  ) => {
+export default class ReceivingAddress extends Component<StateProps & ActionProps, {}> {
+  public onChangeDestinationAddress = (event: React.SyntheticEvent<HTMLInputElement>) => {
     const value = (event.target as HTMLInputElement).value;
     this.props.destinationAddressSwap(value);
   };
 
   public onClickPartTwoComplete = () => {
-    if (!this.props.originAmount) {
+    const { origin, destination } = this.props;
+    if (!origin) {
       return;
     }
     this.props.bityOrderCreateRequestedSwap(
-      this.props.originAmount,
+      origin.amount as number,
```

Comment:
why `as number`? if it has the potential to be null, this doesn't properly fix that case, but does silence the error.